images/release/golang-1.8/Dockerfile

```diff
@@ -31,6 +32,6 @@ RUN mkdir $TMPDIR && \
     git config --system user.name origin-release-container && \
     git config --system user.email none@nowhere.com
 
-WORKDIR /go/src/github.com/openshift/origin
+WORKDIR $WORKDIR
```

Comment:
The main reason I did this was to put WORKDIR to avoid repetition with putting TMPDIR under WORKDIR, since WORKDIR is the directory where the volume is mounted.